.github/workflows/maven.yml

```diff
@@ -7,12 +7,9 @@ on:
   push:
     branches: master
   pull_request:
-    branches: master
 
 jobs:
   build:
-    # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif
-    if: github.repository == 'JacORB/JacORB' && github.event_name == 'pull_request'
```

Comment:
I'd put this in _on purpose_ so that it wasn't running CI on my fork/branches.

Reply:
Hmm, would love to see it run on our fork/branches, will do that locally then